styles/styles.css

```diff
@@ -454,7 +459,7 @@ main .section.background h6 {
 }
 
 .job-description-page p:nth-child(4),
-.job-description-page p:nth-child(8) {
+.job-description-page p:nth-child(7) {
```

Comment:
you can use semantics (like making text italic/strong) so as to avoid the numbering in `p` elements